sonos_websocket/websocket.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         player_id: str | None = None,
         household_id: str | None = None,
         session: aiohttp.ClientSession | None = None,
+        timeout: int | None = None
```

Comment:
Let's just set the default here:
```suggestion
        timeout: int = 6
```